services/views/.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+src/public/javascripts/main.js
```

Comment:
It doesn't look like we have this file so perhaps don't need to include it here?

Reply:
Removed